xapian-core/tests/harness/testsuite.cc

```diff
@@ -706,9 +711,20 @@ test_driver::do_run_tests(vector<string>::const_iterator b,
 		case PASS:
 		    ++res.succeeded;
 		    if (verbose || !use_cr) {
-			out << col_green << " ok" << col_reset << endl;
+			if (test_duration.count() >= 10000) {
+			    out << col_green << " ok" << col_yellow <<
+				" SLOW TEST (" << test_duration.count() <<
+				" ms)" << col_reset << endl;
```

Comment:
The convention for Xapian code when iostream output code is to put the `<<` after the inserted newline.

Reply:
changed accordingly.